crates/bevy_internal/src/default_plugins.rs

```diff
@@ -30,6 +30,8 @@ plugin_group! {
         #[cfg(feature = "bevy_render")]
         bevy_render::texture:::ImagePlugin,
         #[cfg(feature = "bevy_render")]
+        bevy_render::storage:::StoragePlugin,
```

Comment:
Why is this here, and not in the top-level plugin for bevy_render?

Also if you keep it here, there's a list of default plugins somewhere that needs updating to match.

Reply:
This is the list of default plugins, unless you mean something else. I added it here because it's next to the equivalent image asset plugin.

Reply:
Usually we have one plugin per bevy crate that wraps all other plugins in that crate and is used for the default plugin set. I.e. bevy_pbr::PbrPlugin. Not sure why bevy_render is split up though.

Reply:
Ah, it makes sense to me now, it's because it allows easy overriding the `ImagePlugin` at the top level. This isn't the case for this new plugin, so I'll move it into `RenderPlugin`. 👍